kubernetes/job/job.yaml

```diff
@@ -29,6 +29,14 @@ spec:
             requests:
               cpu: "500m"
               memory: "256Mi"
+          livenessProbe:
```

Comment:
Restart policy of the job is defined as:
```
      restartPolicy: "Never"
```

Then why even have a liveness for types Job / CronJob?

Reply:
I check from the https://pkg.go.dev/k8s.io/api/batch/v1 that, job or cronjob both have podspec, so it can have `livenessProbe`. And I think for the cronjob that even the restartPolicy is Never, it need to tell that is it running success or not. What do you think so?

Reply:
@JaneLiuL has a good point. In a cluster with high number of nodes and pods a single descheduler pod can run for minutes. Either a strategy needs more time to decide which pods are candidates for eviction or sending request to the `/pods/evict` subresource is throttled.

Reply:
If liveness probe fails, kubelet will kill the container. However, the job's restart policy is "Never" so it will never re-start.

Reply:
The question here is whether we want to keep an unhealthy pod running or kill it sooner when the liveness probe fails. Even though it's a job. @a7i do you think it is worth taking the case into account?

Reply:
yeah, that's a good point. Let's leave it as-is.

Reply:
It seems we align this comment and not need to update, so may I click resolve conversation?

Reply:
Yes